### PR TITLE
Update webhooks events

### PR DESCRIPTION
**Pull request by mooreds**
Wanted to make the event scope more clear, per feedback from this issue:

https://github.com/FusionAuth/fusionauth-issues/issues/169#issuecomment-858595260

### CODE REVIEW

File: site/docs/v1/tech/events-webhooks/events.adoc [hunk 1]
```diff
@@ -27,13 +27,21 @@ These are the events that FusionAuth generates that can be optionally consumed b
...
+A tenant scoped event can, however contain an `applicationId` which can be used to filter events when received. One example is `user.registration.create`. 
+
 === Application Scoped Events
 
 A few events can be generated for one or more specified applications, or for all applications within a tenant. 
 
 [WARNING]
 ====
-The ability to limit the generation of an event for only certain applications is legacy functionality and may be modified in the future. You almost certainly want to use a tenant scoped event and optionally filter on the `applicationId` when consuming the event.
+The ability to limit the generation of an event for only certain applications is legacy functionality and may be modified in the future. You almost certainly want to enable events at the tenant level and optionally filter on the `applicationId` when consuming the event.
```
Comment (robotdan) [inline site/docs/v1/tech/events-webhooks/events.adoc:44]: > You almost certainly want to enable events at the tenant level and optionally

Not sure what that means. You always need to enable an event at the tenant level if you want to receive it.

Comment (mooreds) [reply site/docs/v1/tech/events-webhooks/events.adoc]: Okay, tried to clarify this.

File: site/docs/v1/tech/events-webhooks/events.adoc [hunk 2]
```diff
@@ -64,10 +62,14 @@ A temporal action is one that has a start time and and a duration, when a phase
...
 
+=== Event Scope
+
+include::docs/v1/tech/events-webhooks/_tenant-or-application-scoped-event.adoc[]
+
 === Event Body
 
 [.api]
-[field]#event.action# [type]#[String]#::
+[field]#event.action# [type]#[String]::
```
Comment (robotdan) [inline site/docs/v1/tech/events-webhooks/events.adoc:72]: Typo? Looks like you didn't mean to remove the trailing `#`

**Review by robotdan — COMMENTED** [review]
looks good

File: site/docs/v1/tech/events-webhooks/events.adoc (hunk 2 above)
Comment (mooreds) [reply site/docs/v1/tech/events-webhooks/events.adoc]: Correct, thanks.

**Pull request merged**